README.md

```diff
@@ -172,8 +172,16 @@ $ bash debian/bin/test-patches ../elan-touchpad-oldkernel.patch 	<<< or whicheve
 $ cd ..
 $ sudo dpkg -i linux-image-4.19.0-4-amd64-unsigned_4.19.28-2a~test_amd64.deb	<<< or whichever you've just compiled
 ```
+## Display
+
+### Fractional Scaling
+On KDE, Deepin and other Qt-based desktop environments, fractional scaling is supported without any hassle but this is not the case for GTK-based desktop environments such as GNOME and XFCE. Well, you can use Wayland but if the app doesn't support Wayland natively, you'll encounter some problems which is fuzziness all over the window. There's a workaround for Xorg:
+Copy `20-intel.conf` and `10-monitor.conf` to `/etc/X11/xorg.conf.d/` and reboot. Voila! You have 150% scaling without any tear - little performance drop.
+I want to add that you shouldn't run your machine with the kernel between 5.0 and 5.5 because one of the kernel maintainer broke something for scaling. They fixed on 5.5. Recommended to use Ubuntu-18.04-HWE kernel because it supports this trick and doesn't have log flooding issue.
```

Comment:
I suggest some rephrasing, to keep in style with the rest of the readme. Also, recommending an Ubuntu kernel for the audience that uses OSes ranging from Fedora and OpenSuse to Linux Mint and Elementary OS sounds a little harsh, doesn't it?

I'd phrase it something along the lines of:

> On KDE, Deepin and other Qt-based desktop environments, fractional scaling is supported without any hassle but this is not the case for GTK-based desktop environments such as GNOME and XFCE. Using Wayland can be an option, but the apps that don't support Wayland natively will still have issues such fuzziness all over the window. A workaround for Xorg allows for 150% scaling without any tear at the const of a little performance drop:
> Copy `20-intel.conf` and `10-monitor.conf` to `/etc/X11/xorg.conf.d/` and reboot.
> It is worth noticing that kernels between 5.0 and 5.5 have known issues with desktop scaling, fixed in 5.5. Ubuntu users can use Ubuntu-18.04-HWE kernel that contains the fix and also is free of the log flooding issue.

Reply:
Also, could you please move the whole Display section higher, between Temperature and Soundcard? I'd love to have it roughly in the same order as the table, so that the table serves as a crude table of contents.

Reply:
Changed